OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -47,13 +47,7 @@
     /**
      * Tells if the page loading has been finished or not.
      */
-    private boolean loadingFinished = true;
-    /**
-     * Url will be considered as redirected if it will not be the initial page url requested to load.
-     * For example, in case the server redirects us to another URL or the user clicks a link
-     * on the web-page, it will be considered as a redirect.
-     */
-    private boolean redirect = false;
+    private boolean loadingFinished = false;
     /**
      * Tells if the currently loading url is the initial page url requested to load.
```

Comment:
Now that we have done quite a bit of discovery on the need for `loadingInitialUrl`, it'll be better to update this description with some technical details/dev notes to avoid extra time wastage on checking various cases to verify the use of this variable.